From 44ef55e81e35fda6d8aae8219bf729b3ad54f84e Mon Sep 17 00:00:00 2001
From: Calvin Johnson <calvin.johnson@nxp.com>
Date: Thu, 4 Oct 2018 09:39:00 +0530
Subject: [PATCH 697/767] staging: fsl_ppfe/eth: resolve indentation warning
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Resolve the following indentation warning:

drivers/staging/fsl_ppfe/pfe_ls1012a_platform.c:
In function ‘pfe_get_gemac_if_proprties’:
drivers/staging/fsl_ppfe/pfe_ls1012a_platform.c:96:2:
warning: this ‘else’ clause does not guard...
[-Wmisleading-indentation]
  else
  ^~~~
drivers/staging/fsl_ppfe/pfe_ls1012a_platform.c:98:3:
note: ...this statement, but the latter is misleadingly indented as
if it were guarded by the ‘else’
   pdata->ls1012a_eth_pdata[port].mdio_muxval = phy_id;
   ^~~~~

Signed-off-by: Calvin Johnson <calvin.johnson@nxp.com>
[Xulin: Original patch taken from NXP LSDK-18.12.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/staging/fsl_ppfe/pfe_ls1012a_platform.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/drivers/staging/fsl_ppfe/pfe_ls1012a_platform.c b/drivers/staging/fsl_ppfe/pfe_ls1012a_platform.c
index f3eb88de66a4..c207f09440f2 100644
--- a/drivers/staging/fsl_ppfe/pfe_ls1012a_platform.c
+++ b/drivers/staging/fsl_ppfe/pfe_ls1012a_platform.c
@@ -89,11 +89,12 @@ static int pfe_get_gemac_if_proprties(struct device_node *parent, int port, int
 	}
 
 	addr = of_get_property(gem, "fsl,mdio-mux-val", &size);
-	if (!addr)
+	if (!addr) {
 		pr_err("%s: Invalid mdio-mux-val....\n", __func__);
-	else
+	} else {
 		phy_id = be32_to_cpup(addr);
 		pdata->ls1012a_eth_pdata[port].mdio_muxval = phy_id;
+	}
 
 	if (pdata->ls1012a_eth_pdata[port].phy_id < 32)
 		pfe->mdio_muxval[pdata->ls1012a_eth_pdata[port].phy_id] =
-- 
2.17.0

